components/TVChart/TVChart.tsx

```diff
@@ -82,17 +82,8 @@ export function TVChart({
 		return () => {
 			clearExistingWidget();
 		};
-	}, [
-		baseCurrencyKey,
-		quoteCurrencyKey,
-		autosize,
-		containerId,
-		fullscreen,
-		interval,
-		libraryPath,
-		overrides,
-		studiesOverrides,
-	]);
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, []);
```

Comment:
I think we need the `baseCurrencyKey` and `quoteCurrencyKey` in the deps array here (only these two). Sorry for the confusion.

Reply:
will fix